README.md

```diff
@@ -1,2 +1,174 @@
-# react-capstone-metrics
-A capstone that depicts information about countries via a RESTful API. Their population, landmarks, area, longitude and latitude and a host of others.
+<div align="center">
+  <h3><b>React Capstone Metrics</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖React Capstone Metrics <a name="about-project"></a>
+
+**A capstone that depicts information about countries via a RESTful API. Their population, landmarks, area, longitude and latitude and a host of others.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>API</summary>
+  <ul>
+    <li><a href="https://restcountries.com/">Country Api</a>a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Show countries information**
+- **Displays continent information**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo](soon)
```

Comment:
- [ ] Kindly ensure you deploy your app and provide its link in the README.

  ![image](https://github.com/Theodoraldo/react-capstone-metrics/assets/36542986/4ebf72f3-2ebd-44c4-b817-cf1fbe408f6e)

- [ ] Kindly record a video presentation of your app and include the link in the README. Ensure you outline the piece of code you are proud of or find interesting.